### ISSUE TITLE
Wake Word detection responses resulting in: Assist resposne saying "I dont understand"

### ISSUE THREAD

**Issue by afnieves** [relust/VisualStreamAssist#2]
Wake Word detection responses are being included in the STT

Sentence:

Wake Word         Voice Prompt                   Requested Action
"OK Nabu"          How Can I help you.       Turn on bedroom light

Results:

I dont under stand <------ 

RAW:

"How Can I help you. Turn on bedroom light"

Is there a way to turn off MIC during the Wake Word detection responses so they dont get sent as part of the requsted action?
Or maybe strip the Wake Word detection responses from the request ?

**Comment by relust** [comment]
On the android tablets I use, samsung and lenovo, the tablet microphone does not record the custom responses that are played on the tablet speaker. When I tried on other devices such as an ip camera and a google home speaker, I found this problem that I also record these custom responses, and so far I have not found a way to turn off the microphone or introduce a delay between detection wake word and effective listening. If it didn't work, I recommend you to use an esp32 satellite for which I managed to make a mute switch. I asked Mike how he did this on the satellites with the assist microphone addon and the raspberry pi satellites and he told me that these satellites have an echo ignore software to not record the surrounding noises like the TV, and that ignore these custom responses.

**Comment by afnieves** [comment]
Can i use a m5stack-atom-echo next to the tablet?  If so, how would that be configured in Visual Stream Assist ?

**Comment by relust** [comment]
Yes, you can use m5stack but not with visual stream assist integration. You can use this project for esp32 sattelites https://github.com/relust/HA-Visual-Voice-Assistant
